src/stdlib.jl

```diff
@@ -1,7 +1,15 @@
 import UUIDs
 
+if VERSION < v"1.4"
+    stdlibs = Pkg.Types.stdlib()
+else
+    # renamed to stdlibs
+    # https://github.com/JuliaLang/Pkg.jl/pull/1559
+    stdlibs = Pkg.Types.stdlibs()
+end
+
 # This function is based off of a similar function here:
 # https://github.com/JuliaRegistries/RegistryCI.jl/blob/master/src/RegistryCI.jl
 function gather_stdlib_uuids()
-    return Set{UUIDs.UUID}(x for x in keys(Pkg.Types.stdlib()))
+    return Set{UUIDs.UUID}(x for x in keys(stdlibs()))
```

Comment:
You need the two changes above, or the following:
```suggestion
    return Set{UUIDs.UUID}(x for x in keys(stdlibs))
```

Reply:
Ooops my mistake.